chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/MetaDataRenderer.java

```diff
@@ -242,7 +239,7 @@ public boolean isDrawOnCanvas() {
     }
 
     @Override
-    public void render(final GraphicsContext gc, final Chart chart, final int dataSetOffset,
+    public List<DataSet> render(final GraphicsContext gc, final Chart chart, final int dataSetOffset,
```

Comment:
Codacy found an issue: [The method 'render(GraphicsContext, Chart, int, ObservableList)' has an NPath complexity of 512](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5764587)